### PR TITLE
Fix testing

### PR DESCRIPTION
**Pull request by jennmald**
1. Add the status codes for verifying connection
2. Fix a key issue in response json

### CODE REVIEW

File: deep_beamline_simulation/tests/test_ophyd_attempt.py [hunk 1]
```diff
@@ -6,9 +6,10 @@
 
 def test_connection():
     session = requests.Session()
-    response_sim_list = session.post("http://localhost:8000/simulation-list", json={"simulationType": "srw"})
-    expected = '<Response [200]>'
-    assert str(response_sim_list) == expected
+    response_sim_list = session.post(
+        "http://localhost:8000/simulation-list", json={"simulationType": "srw"}
+    )
+    assert response_sim_list.status_code == 200
```
[the remark this answers is not in the document]
Comment (jennmald) [reply deep_beamline_simulation/tests/test_ophyd_attempt.py:12]: yes! I verify data in all other tests

File: deep_beamline_simulation/tests/test_ophyd_attempt.py [hunk 2]
```diff
@@ -153,43 +154,41 @@ def test_find_sim_by_name():
...
+    expected_response["simulation"].pop("lastModified")
     expected_response["simulation"].pop("simulationId")
     expected_response["simulation"].pop("simulationSerial")
     # remove changing variables for actual response
     actual_response.pop("simulationId")
-    actual_response.pop("last_modified")
-    actual_response["simulation"].pop("outOfSessionSimulationId")
+    actual_response["simulation"].pop("lastModified")
     actual_response["simulation"].pop("simulationId")
     actual_response["simulation"].pop("simulationSerial")
```
[the remark this answers is not in the document]
Comment (jennmald) [reply deep_beamline_simulation/tests/test_ophyd_attempt.py:193]: Do you suggest a loop checking both responses and popping which ones differ?

File: deep_beamline_simulation/tests/test_ophyd_attempt.py [hunk 3]
```diff
@@ -153,43 +154,41 @@ def test_find_sim_by_name():
...
             "verticalPosition": 0,
             "verticalRange": 1.3,
         },
     }
     expected_response.pop("simulationId")
-    expected_response.pop("last_modified")
-    expected_response["simulation"].pop("outOfSessionSimulationId")
+    expected_response["simulation"].pop("lastModified")
     expected_response["simulation"].pop("simulationId")
     expected_response["simulation"].pop("simulationSerial")
```
[the remark this answers is not in the document]
Comment (jennmald) [reply deep_beamline_simulation/tests/test_ophyd_attempt.py:188]: done!

**Comment by jennmald** [comment]
1. Create function for repeating lines of code in `test_runSimulation` and `test_simulationUpload`
2. Fixed identical if statements by combining into one
3. Added assert to `test_runSimulation` to verify summary data
4. Fixed `tensorfile.py` to `readlines()` from `readline()`
5. Added `for` loop to pop differing values for testing
6. Fixed status codes

To Do: 

- Dynamic parameter exclusion
- Move next PR to a feature branch like this one

File: deep_beamline_simulation/tests/test_ophyd_attempt.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mrakitin) [reply deep_beamline_simulation/tests/test_ophyd_attempt.py]: I am thinking about a list of keywords to exclude from each dictionary before comparison, such as:
```py
fields_to_exclude = ["lastModified", "simulationId", "simulationSerial"]
for field in fields_to_exclude:
    expected_response["simulation"].pop(field)
    actual_response["simulation"].pop(field)
```
That way you'll be comparing the dictionaries without those varying fields.

[the remark this answers is not in the document]
Comment (jennmald) [reply deep_beamline_simulation/tests/test_ophyd_attempt.py]: thanks! I think we had the same thing in mind. When you get the chance to review the new commits I just put in you'll see I have the identical code.

**Review by mrakitin — APPROVED** [review]
Thanks for addressing my suggestions, @jennmald!

**Review by jklynch — APPROVED** [review]

**Pull request merged**